authors/settings.py

```diff
@@ -170,4 +170,12 @@
         },
     },
 }
- 
\ No newline at end of file
+
+
+SECRET_KEY = 'this is our secret'
```

Comment:
The contents of this should be a secret.

Reply:
I have added that